app/src/test/java/com/github/multimatum_team/multimatum/FragmentProfileTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.multimatum_team.multimatum
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.firebase.FirebaseApp
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.ktx.initialize
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+
+class FragmentProfileTest {
+    @get:Rule
+    val activityRule = ActivityScenarioRule(AccountActivity::class.java)
+
+
+    //Impossible to test
+   /* @Test
+    fun `lauch intent when click logout button`(){
```

Comment:
Why this does not work ?

Reply:
because i need a firebaseApp running in a android app context and the test is not. And it's seems pretty impossible to create this context in the test.